js/location.js

```diff
@@ -0,0 +1,25 @@
+//if user blocks the location the browser will show the weather in Arvada, CO
+//post weather pin by location
+if(window.screen.width >= 800) {
+navigator.geolocation.getCurrentPosition(function (position) {
+    latitude = position.coords.latitude.toFixed(2)
+    longitude = position.coords.longitude.toFixed(2)
+    
+    console.log(`latitude: ${latitude}`)
```

Comment:
remove console.logs if you are done.

Reply:
Thank you very much for your review. Really appreciate this. Will follow all your advices